docs/current_docs/configuration/cache.mdx

```diff
@@ -0,0 +1,37 @@
+---
+slug: /configuration/cache
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Cache
+
+Dagger caches two types of data:
+
+1. Layers: This refers to build instructions and the results of some API calls. This cache is implemented by Buildkit.
+2. Volumes: This refers to the contents of a Dagger filesystem volume and is persisted across Dagger Engine sessions. It is implemented by Dagger (distinct from Buildkit).
```

Comment:
> It is implemented by Dagger (distinct from Buildkit).

Cache volumes are still implemented by buildkit.

Reply:
+1

Reply:
So both caches are managed by BuildKit? In this case, I propose to remove the BuildKit reference altogether, as I'm not sure naming the underlying tech is useful to an end-user.